api/namex/services/name_request/name_request.py

```diff
@@ -144,7 +144,7 @@ def extend_expiry_date(cls, name_request, start_date=None, days=56):
         """
         Extends/sets the expiry date of an NR.
 
-        Default: extends the expiry date to 56 days from tomorrow's date at 00:01am.
+        Default: extends the expiry date to 56 days date at 11:59pm.
         """
         start_datetime = start_date if start_date else datetime.utcnow()
```

Comment:
Does this need the same fix as above? I'm concerned about the possible assignment of a UTC date.

Reply:
I don't think we need to change it here. What are you comparing with? (above?)

Reply:
It liks like it can set start_datetime as "now" in UTC. Should that be "now" local?

Reply:
As it is timezone aware it is ok. If you create 2 datetimes: one with Pacifc time and another in UTC with the same datetime, both when converted to the same timezone should have the same values.

Reply:
OK